modules/reports/server-ts/sql.ts

```diff
@@ -0,0 +1,10 @@
+import { knex } from '@gqlapp/database-server-ts';
+
+export default class Report {
+  public report() {
```

Comment:
It looks like you selecting all data, then it should have name `reports` or even `users` because you selecting users from `report` table. @evgenijolshanskij  @vlasenko  what do you think about this?

Reply:
@lyzhovnik Should be named `getUsers` here I think

Reply:
The DB table name should reflect the data it contains. Despite the name of the module, the table and DAO class are needed to be renamed (`contacts`, `users` etc.).

Reply:
I'd use different from `user` name for not confusing anyone. Just make up some other entity.

Reply:
Fixed in #1023